Timeline/src/[sandbox]/sandbox.ts

```diff
@@ -140,8 +139,8 @@ export class Sandbox extends LitElement {
               person-id="30313-Carl"
               show-filters
               base-url="https://cjaas-devus2.azurewebsites.net"
-              .stream-read-token=${getSToken()}
-              .tape-read-token=${getTToken()}
+              tape-read-token="so=demoassure&sn=sandbox&ss=tape&sp=r&se=2022-11-23T20:33:44.019Z&sk=journeyUi&sig=Msa4zTsNmkeDHJcmQuXUVHTTzs1KATCQ%2FDNrVR2O7eU%3D"
+              stream-read-token="so=demoassure&sn=sandbox&ss=stream&sp=r&se=2022-11-23T20:30:20.765Z&sk=journeyUi&sig=76cI1nBPkA0HdQved8YHiTQbOThPOR8W5UdwZzeUuPc%3D"
```

Comment:
why did you remove the dot notation in front of each property?

Reply:
with dot notation it should have been `.tapeReadToken` We dint need it with static text.

The SAS tokens generated in browser was not valid for some reason.